plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/internal/RegistryMessages.properties

```diff
@@ -17,3 +17,9 @@ navigator_settings_preset_advanced_view_description = Shows all database objects
 navigator_settings_preset_custom_view_name = Custom
 navigator_settings_preset_custom_view_description = User configuration
 project_description_comment = General DBeaver project
+
+project_open_cannot_read_credentials_title = You are about to lose project credentials
+project_open_cannot_read_credentials_message = Your are about opening encrypted project: ''{0}''\n\nWarning: if you proceed all your credentials will be permanently deleted. To avoid losing it please open project in DBeaver PRO version.
```

Comment:
```suggestion
project_open_cannot_read_credentials_message = You are about to open an encrypted project: ''{0}''\n\nWarning: if you proceed, all your credentials will be permanently deleted. To avoid losing them, please open your project in DBeaver PRO version.
```